release_notes/ols-release-notes.adoc

```diff
@@ -13,6 +13,8 @@ The release notes highlight what is new and what has changed with each {ols-offi
 {ols-official} is designed for Federal Information Processing Standards (FIPS). When running on {ocp-product-title} in FIPS mode, it uses the {rhel} cryptographic libraries submitted (or planned to be submitted) to NIST for FIPS validation on only the `x86_64`, `ppc64le`, and `s390X` architectures. For more information about the NIST validation program, see link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic Module Validation Program]. For the latest NIST status of the individual versions of {rhel} cryptographic libraries that have been submitted for validation, see link:https://access.redhat.com/en/compliance[Product compliance].
 ====
 
+include::modules/ols-1-0-3-release-notes.adoc[leveloffset=+1]
+include::modules/ols-1-0-3-fixed-issues.adoc[leveloffset=2]
```

Comment:
it is a bit odd to see absolute offset `leveloffset=2` instead of `leveloffset=+2` but it is ok here - just a suggestion for the future